packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -43,7 +43,9 @@ export type Props = {
   tagline: ?string,
   headline: string,
   subheading: ?string,
-  image: string,
+  backgroundImageMobile: string,
+  backgroundImageTablet: string,
+  backgroundImageDesktop: string,
```

Comment:
I am nitpicking here, but just because we have too many properties in the top-level, can we nest these images under an `image` property, as suggested [here](https://confluence.skyscannertools.net/display/SRVT/Front-End+Work+for+Graphic+Promos)? Feel free to use `desktop`, `tablet` and `mobile`, if these are the agreed variations instead of `landscape` and `portrait`.

Thanks! 😄

Reply:
Resolving since we're no longer passing through these props